concrete/single_pages/dashboard/system/permissions/maintenance.php

```diff
@@ -7,7 +7,8 @@
     <?php echo $this->controller->token->output('update_maintenance')?>
     <fieldset>
         <div class="form-group">
-            <div class="radio"><label><?=$form->radio('site_maintenance_mode', '1', $site_maintenance_mode)?> <span><?=t('Enabled')?></span></label></div>
+            <label><?= t('Only Dashboard Works') ?></label>
```

Comment:
@mesparza5 Since this is a Bootstrap 3 label, we should be using <label class="control-label"> instead of just a label tag with no class.